Views/Lombiq.Privacy.ConsentBanner.cshtml

```diff
@@ -1,67 +1,87 @@
+    <div class="modal-content">
+        <div class="modal-header bg-secondary">
+            <h5 class="modal-title" id="exampleModalLabel">@T["Privacy consent"]</h5>
+        </div>
+        <div class="modal-body">
+            <shape Type="Lombiq_Privacy_ConsentBanner_Text" />
+        </div>
+        <div class="modal-footer justify-content-center">
+            <div class="btn-group btn-group-lg" role="group">
+                <button id="@acceptButtonId" type="button" class="btn btn-success" data-cookie-string="@acceptCookie">
+                    @T["I agree, do not show me this any more"]
+                </button>
+                @if (!isAuthenticatedHasNotConsent)
+                {
+                    <button id="@deniedButtonId" type="button" class="btn btn-secondary" data-cookie-string="@deniedCookie">
+                        @T["I don't agree, do not show me this any more"]
+                    </button>
+                }
+            </div>
+        </div>
+    </div>
 </div>
 
+@if (!isAuthenticatedHasNotConsent)
+{
+    <script depends-on="jQuery" at="Foot">
+        $("#@acceptButtonId").click(function () {
+            document.cookie = $("#@acceptButtonId")[0].dataset.cookieString;
+            $("#@bannerId").remove();
+        });
 
-@using (Script.Foot())
+        $("#@deniedButtonId").click(function () {
+            document.cookie = $("#@deniedButtonId")[0].dataset.cookieString;
+            $("#@bannerId").remove();
+        });
+    </script>
+}
+else
 {
-    <script type="text/javascript">
-        ; (function ($) {
-            $(function () {
-                if (!ConsentCookie.HasConsentCookie()) {
-                    $("#@ConsentBannerId").dialog({
-                        closeOnEscape: false,
-                        resizable: false,
-                        draggable: false,
-                        position: {
-                            my: "right-10% top+10",
-                            at: "right top+50",
-                            of: "html"
-                        },
-                        open: function (event, ui) {
-                            jQuery('.ui-dialog-titlebar-close').hide();
-                        },
-                        buttons: {
-                            "@T("I agree, do not show me this any more").Text": function () {
-                                ConsentCookie.Set();
-                                $(this).dialog("close");
-                            },
-                            "@T("I agree, but continue to show me this each time I visit the site").Text": function () {
-                                ConsentCookie.Set("true");
-                                $(this).dialog("close");
-                            }
-                        }
-                    });
-                }
-            });
-        })(jQuery);
+    var request = Context.Request;
+
+    var acceptanceOfConsentURL = new UriBuilder

```

Comment:
Elég csak relatív URL-t használni az `Url.Action()`-nel.

Reply:
Eredetileg úgy volt de próbáltam például a /Admin/Features elvégezni ezt a requestet és ebben az esetben relativ url esetén hozzárakta a /Admint is. De letesztelem mégegyszer a biztonság kedvért